### PR TITLE
add build_and_test_foxy and build_and_test_galactic to rolling branch

### LINKED ISSUE #28: CI badges aren't showing up correctly
CI badges for Galactic and Foxy aren't showing up correctly. I think the issue may be that github actions needs the CI to be available on the main branch to use the badges. The Foxy and Galactic actions don't show up in the Actions tab in this repository.

### PR DESCRIPTION
**Pull request by ijnek**
Add ``build_and_test_foxy.yaml`` and ``build_and_test_galactic.yaml`` to the default rolling branch. I removed these two files when I split off the galactic branch, but it seems like they are necessary for Github to recognise the actions even exist.

Resolves: #28

Signed-off-by: Kenji Brameld <kenjibrameld@gmail.com>

### CODE REVIEW

**Review by SammyRamone — APPROVED** [review]

**Comment by ijnek** [comment]
Could one of the maintainers (@Flova or @jaagut) approve too?

**Review by jaagut — APPROVED** [review]

**Comment by ijnek** [comment]
Confirmed the badges are showing up correctly after the merge.

**Pull request merged**
